### PR TITLE
Resolves Issue 126

### PR DESCRIPTION
**Pull request by HeinzBaumann**
This PR addresses several issues found in the EEA Section spec and brings this spec to date with the latest TCF 2.2 specification.

### CODE REVIEW

File: Sections/EEA/GPPExtension: IAB Europe TCF.md [hunk 1]
```diff
@@ -129,30 +135,48 @@ The core segment must always be present. It consists of the following fields:
...
     <tr>
+  <tr style="background-color:#000;color:#FFF;">
+    <td colspan="4">
+        <strong>Specific Jurisdiction Disclosures</strong>
+    </td>
+  </tr>
   <td>PurposeOneTreatment</td>
   <td>Boolean</td>
-  <td>1 Purpose 1 was NOT disclosed at all.<br><br>0 Purpose 1 was disclosed commonly as consent as expected by the Policies.<br><br>CMPs can use the PublisherCC field to indicate the legal jurisdiction the publisher is under to help vendors determine whether the vendor needs consent for Purpose 1.</td>
+  <td>1 Purpose 1 was NOT disclosed at all.<br>0 Purpose 1 was disclosed commonly as consent as expected by the Policies.<br>CMPs can use the PublisherCC field to indicate the legal jurisdiction the publisher is under to help vendors determine whether the vendor needs consent for Purpose 1.</td>
```
Comment (janwinkler) [inline Sections/EEA/GPPExtension: IAB Europe TCF.md:145]: can we rephrase the logic of this. having a 1 as "no" and 0 as "yes" seems incorrect and may lead to errors. better something like:
0 = No special treatment. Purpose 1 was disclosed
1 = Special treatment. Purpose 0 was not disclosed

File: Sections/EEA/GPPExtension: IAB Europe TCF.md [hunk 2]
```diff
@@ -129,30 +135,48 @@ The core segment must always be present. It consists of the following fields:
...
   </tr>
   <tr>
   <td>PublisherCC</td>
   <td>String (2)</td>
-  <td>The country code of the country that determines legislation of reference. Commonly, this corresponds to the country in which the publisher’s business entity is established.
-    Each letter is encoded as 6 bits, <code>a=0..z=25</code>. <a href="https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2"> ISO 3166-1 alpha-2 code </a> </td>
+    <td>The country code of the country that determines legislation of reference. Commonly, this corresponds to the country in which the publisher’s business entity is established.
```
Comment (janwinkler) [inline Sections/EEA/GPPExtension: IAB Europe TCF.md:150]: just for sake of clarity lets add "lowercase" to it

File: Sections/EEA/GPPExtension: IAB Europe TCF.md [hunk 3]
```diff
@@ -129,30 +135,48 @@ The core segment must always be present. It consists of the following fields:
...
+    </td>
+  </tr>
+  <tr style="background-color:#000;color:#FFF;">
+  <td colspan="4">
+        <strong>Vendor Consent and Vendor Legitimate Interest Sections</strong>
+      </td>
   </tr>
   <tr>
-  <td>VendorConsent</td>
+  <td>VendorConsent*</td>
```
Comment (janwinkler) [inline Sections/EEA/GPPExtension: IAB Europe TCF.md:160]: remove the *, the field will be returned by the API

File: Sections/EEA/GPPExtension: IAB Europe TCF.md [hunk 4]
```diff
@@ -129,30 +135,48 @@ The core segment must always be present. It consists of the following fields:
...
+  <td>List of Vendor IDs that indicate Consent for these vendors. Please note that the field is composed of several fields in order to store the data in GPP string format. The client side API format is an array of int.<br><br>Equivalent of the fields listed under the Vendor Consent Section as defined in the <a href="https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md">TCF v2 specification:</a> <br>MaxVendorId, <br>IsRangeEncoding,<br>BitField, NumEntries, IsARange, StartOrOnlyVendorId, EndVendorId</td>
   </tr>
   <tr>
-  <td>VendorLegitimateInterest</td>
+  <td>VendorLegitimateInterest*</td>
```
Comment (janwinkler) [inline Sections/EEA/GPPExtension: IAB Europe TCF.md:165]: remove the *, the field will be returned by the API

File: Sections/EEA/GPPExtension: IAB Europe TCF.md [hunk 5]
```diff
@@ -129,30 +135,48 @@ The core segment must always be present. It consists of the following fields:
...
+      <strong>Publisher Restrictions Section</strong>
+    </td>
+    <td colspan=2>The content of this section is optional EXCEPT for NumPubRestrictions. Encodes any number of single or range restriction entries.
+    </td>
   </tr>
   <tr>
   <td>NumPubRestrictions</td>
   <td>Int(12)</td>
-  <td>Number of restriction records to follow.<br></br>Value is required even if it is <code>0</code></td>
+  <td>Number of restriction records to follow.<br>Value is required even if it is <code>0</code></td>
```
Comment (janwinkler) [inline Sections/EEA/GPPExtension: IAB Europe TCF.md:179]: needs to be removed and replaced with one field of type ArrayOfRanges (replaces NumPubRestrictions, PurposeID, RestrictionType, PubRestrictionEntry)

File: Sections/EEA/GPPExtension: IAB Europe TCF.md [hunk 6]
```diff
@@ -25,16 +25,22 @@ The global standard [GPP](https://github.com/InteractiveAdvertisingBureau/Global
...
+Note that the resulting TC String produced is expected to be exactly the same as the TC String produced when using the [TCF specification](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md). For full details about the TC String, please refer to the [Consent string and vendor list format v2](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md).
+
+Note: In the event of any discrepancies between this GPP section specification and the [TCF specifications](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework), the TCF specifications take precedence and must be followed.
```
[the remark this answers is not in the document]
Comment (HeinzBaumann) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md:38]: reviewed by FSWG. We leave the current text.

File: Sections/EEA/GPPExtension: IAB Europe TCF.md [hunk 7]
```diff
@@ -90,12 +96,12 @@ The core segment must always be present. It consists of the following fields:
...
+    <td>1 true<br>0 false<br>This field must always have the value of 1. When a Vendor encounters a TC String with IsServiceSpecific=0 then it is considered invalid.</td>
   </tr>
   <tr>
-  <td>UseNonStandardStacks</td>
+  <td>UseNonStandardTexts</td>
   <td>Boolean</td>
-  <td>1 CMP used non-IAB standard stacks during consent gathering<br><br>0 IAB standard stacks were used<br><br> Setting this to 1 means that a publisher-run CMP – that is still IAB Europe registered – is using customized Stack descriptions and not the standard stack descriptions defined in the <a href="https://iabeurope.eu/iab-europe-transparency-consent-framework-policies/">Policies</a> (Appendix A section E). A CMP that services multiple publishers sets this value to 0.</td>
+  <td>Setting this to 1 signals to Vendors that a private CMP has modified standard Stack descriptions and/or their translations and/or that a CMP has modified or supplemented standard Illustrations and/or their translations as allowed by the <a href="https://iabeurope.eu/iab-europe-transparency-consent-framework-policies/">policy</a> (Appendix A section E). A CMP that services multiple publishers sets this value to 0.</td>
```
[the remark this answers is not in the document]
Comment (HeinzBaumann) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md:104]: Will remove the private vs. public CMP from this paragraph,. Also update the TCF spec accordingly.

File: Sections/EEA/GPPExtension: IAB Europe TCF.md (hunk 1 above)
Comment (HeinzBaumann) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md:145]: Update accordingly here and in the TCF specification.

File: Sections/EEA/GPPExtension: IAB Europe TCF.md (hunk 2 above)
Comment (HeinzBaumann) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md:150]: Declined by FSWG

File: Sections/EEA/GPPExtension: IAB Europe TCF.md (hunk 4 above)
Comment (HeinzBaumann) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md]: Fixed

File: Sections/EEA/GPPExtension: IAB Europe TCF.md (hunk 5 above)
Comment (HeinzBaumann) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md]: Fixed, @janwinkler please take a look as the new section.

File: Sections/EEA/GPPExtension: IAB Europe TCF.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (HeinzBaumann) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md]: Fixed

File: Sections/EEA/GPPExtension: IAB Europe TCF.md (hunk 1 above)
Comment (HeinzBaumann) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md]: Fixed

File: Sections/EEA/GPPExtension: IAB Europe TCF.md (hunk 5 above)
Comment (janwinkler) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md]: @HeinzBaumann maybe just clarification on the last line:
"A single or range of Vendor ID(s) who the publisher has designated as restricted under the purpose id."
--> "A list of Vendor ID(s) who the publisher has designated as restricted under the purpose id encoded as bitmask or ranges."

Comment (HeinzBaumann) [reply Sections/EEA/GPPExtension: IAB Europe TCF.md]: Thanks! I made this modification.

**Review by janwinkler — APPROVED** [review]

**Review by schavvaOt — APPROVED** [review]

**Review by dev-spandey — APPROVED** [review]

**Pull request merged**